.kokoro

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+if [ ! -d .kokoro-ios-runner ]; then
+  git clone https://github.com/material-foundation/kokoro-ios-runner.git .kokoro-ios-runner
+fi
+
+pushd .kokoro-ios-runner
+git fetch > /dev/null
+git checkout v3.1.0 > /dev/null
```

Comment:
Not HEAD?
Poor shell-fu to get latest major version 3?
TAG=`git tag -l "v3*" | tail -n1`
git checkout $TAG > /dev/null

Reply:
Done.